### DISCUSSION TITLE
Is it possible to Update Dockerimages in README or other Markdown files?

### DISCUSSION THREAD

**Discussion by mstoetzer** [renovatebot/renovate#9149] [Request Help] [answered]
As the title says, I want to know, if it's possible to configure renovate to update Docker images in Markdown files.

E.g. a README.md contains instructions what to copy into the shell window or how to use a container image manually for testing. Would be super cool, if renovate could also take care about those markdown files. Is this possible?

Thank you.

**Comment by rarkins** [comment, marked as answer]
Yes, we do it for Renovate's markdown files. See the [regex manager](https://github.com/renovatebot/.github/blob/cf278491c92f78960db271c34d58522051aedfc3/default.json#L96-L104) and the [package rule](https://github.com/renovatebot/.github/blob/cf278491c92f78960db271c34d58522051aedfc3/default.json#L43-L54).

Note that it's updating a specific image we have specified, it is not "discovering" random Docker images.

**Comment by mstoetzer** [comment]
Thank you! I'll try this out :)
